src/Semantics.js

```diff
@@ -369,8 +370,9 @@ function newDefaultAction(type, name, doIt) {
     } else {
       // Otherwise, we throw an exception to let the programmer know that we don't know what
       // to do with this node.
-      throw new Error(
-          'Missing semantic action for ' + this.ctorName + ' in ' + name + ' ' + type);
+
+      throw errors.semanticErrors(
+          this.ctorName, name, type) ;
```

Comment:
If we don't throw the error and just return the error, then how should the error message/stack be printed/handled?